src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-details/rbd-details.component.ts

```diff
@@ -20,8 +19,8 @@ export class RbdDetailsComponent implements OnChanges {
   constructor() {}
 
   ngOnChanges() {
```

Comment:
```suggestion
```

Reply:
I guess I cannot remove ngOnChanges, otherwise 
```
  <cd-rbd-details cdTableDetail
                  [selection]="expandedRow">
  </cd-rbd-details>
```
would fail in rbd-lst.component.html